marshall/gulpfile.js

```diff
@@ -1,5 +1,16 @@
 var gulp = require('gulp');
+var sass = require('gulp-eslint');
+var watch = require('gulp-watch');
 
-gulp.task('default', function() {
-  
+gulp.task('default', ['lint', 'test'], function(){
+    console.log('GULP LIVES!!!')
 });
+
+gulp.task('lint', function() {
+ return gulp.src(['./greet.js', '!node_modules/gulp-eslint'])
+  .pipe(eslint());
+  .pipe(eslint.format());
+  .pipe(eslint.failAfterError());
+});
```

Comment:
Some syntax error on each of these lines is preventing you from using pipe.  See if you can figure out what it is.